README.md

```diff
@@ -566,6 +566,12 @@ These are available from the `@hyperjump/json-schema/experimental` export.
     `$vocabulary` keyword in the meta-schema. The only time you would need to
     load a dialect manually is if you're creating a distinct version of JSON
     Schema rather than creating a dialect of an existing version of JSON Schema.
+* **getDialectIds**: () => string[]
+    
+    This function retrieves the identifiers of all loaded JSON Schema dialects.
+    This function provides a convenient way to retrieve the identifiers of all loaded JSON Schema dialects,
+    enabling further customization or querying based on the available dialects.
+    It returns an array of strings containing the identifiers of all loaded JSON Schema dialects
```

Comment:
This feels like it's saying the same thing three times. Let's simplify. The last sentence is unnecessary because the types already communicate that it returns an array of strings. We could keep the first or the second sentence, but I don't think we need both. I prefer the first.

```suggestion
    This function retrieves the identifiers of all loaded JSON Schema dialects.
```